hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/multipart/S3MultipartUploadCompleteRequest.java

```diff
@@ -472,8 +472,10 @@ private long getMultipartDataSize(String requestedVolume,
           .getKeyLocationVersions().get(0);
 
       // Set partNumber in each block.
-      currentKeyInfoGroup.getLocationList().forEach(
-          omKeyLocationInfo -> omKeyLocationInfo.setPartNumber(partNumber));
+      currentKeyInfoGroup.getLocationLists().stream()
+          .flatMap(List::stream)
+          .forEach(omKeyLocationInfo ->
+              omKeyLocationInfo.setPartNumber(partNumber));
 
       partLocationInfos.addAll(currentKeyInfoGroup.getLocationList());
```

Comment:
There is another call to `getLocationList` here.  If we keep it, might as well store the list and reuse it for setting part number, too.